modules/dashboard/locals.tf

```diff
@@ -60,12 +70,11 @@ locals {
       height            = 5
       expressions       = []
       yAxis             = { left = { min = 0 } }
-      data_source       = var.data_source
+      datasource        = var.data_source
```

Comment:
why we changed data_source => datasource here, I don't see there another data_source => datasource  change to get this change impacted on existing codes, we seems still use data_source everywhere in old codes and only new codes have datasource way?
and it seems because of this everywhere in code we have to use `try(each.value.data_source, {})` construction,